.github/workflows/release.yml

```
@@ -38,7 +51,7 @@ jobs:
 
       # Upload using our oldest, tested resolver, to set the lowest lower bounds
       - name: "Release to Hackage"
-        run: stack --stack-yaml stack-lts-20.26.yaml upload --pvp-bounds lower freckle-app
+        run: stack --stack-yaml stack-lts-20.26.yaml upload --pvp-bounds lower ${{ matrix.package }}
 
       - name: "Upload documentation"
-        run: stack upload --documentation freckle-app
+        run: stack upload --documentation ${{ matrix.package }}
```

This isn't normally necessary, it was added because Hackage can't build with the librdkafka dependency. That means I think it can be:

```suggestion
        if: ${{ matrix.package == 'freckle-kafka' }}
        run: stack upload --documentation ${{ matrix.package }}
```

Alternatively, you could add something like,

```yaml
include:
  - package: freckle-kafka
    upload-docs: true
```

Then make this `matrix.upload-docs == 'true'`. That would be useful if we think additional packages (but still not all) will need this step, but I think that's pretty unlikely.